boards/common/nrf51/Makefile.include

```diff
@@ -0,0 +1,22 @@
+# define the used CPU
+export CPU = nrf51
+export CPU_MODEL ?= nrf51x22xxaa
+
+# define the default port depending on the host OS
+PORT_LINUX ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# most of nrf51 boards use an ST-Link v2 debug adapter
+export DEBUG_ADAPTER ?= stlink
+export STLINK_VERSION ?= 2
+
+# include this module into the build
+INCLUDES += -I$(RIOTBOARD)/common/nrf51/include
+USEMODULE += boards_common_nrf51
+
+# use common openocd configuration for nrf51
```

Comment:
only needed in case openocd is actually used as a programmer. So does it make sense to ifdef this based on the used programmer?